stable/mariadb/templates/master-statefulset.yaml

```diff
@@ -26,7 +26,7 @@ spec:
       {{- if .Values.master.annotations }}
       annotations:
         {{- range $key, $value := .Values.master.annotations }}
-        {{ $key }}: {{ $value }}
+        {{ $key }}: {{ $value | quote }}
```

Comment:
if $value is an array (just like the example you mentioned) what would the output look like?

Reply:
currently we have this in our values.yaml

```yaml
mariadb:
  master:
    annotations:
      pre.hook.backup.velero.io/command: '["sh", "-c", "/opt/bitnami/mariadb/bin/mysqldump -A -u root -p$MARIADB_ROOT_PASSWORD | /bin/gzip > /bitnami/mariadb/full_dump.gz"]'
```

this results to:
```yaml
apiVersion: apps/v1
kind: StatefulSet
metadata:
  name: ***
  labels:
    app: mariadb
    chart: mariadb-7.3.4
    release: ***
    heritage: Helm
    component: master
spec:
  selector:
    matchLabels:
      app: mariadb
      release: ***
      component: master
  serviceName: ****
  replicas: 1
  updateStrategy:
    type: RollingUpdate
  template:
    metadata:
      annotations:
        backup.velero.io/backup-volumes: data
        pre.hook.backup.velero.io/command: ["sh", "-c", "/opt/bitnami/mariadb/bin/mysqldump -A -u root -p$MARIADB_ROOT_PASSWORD | /bin/gzip > /bitnami/mariadb/full_dump.gz"]
```

so the string gets convertet to a yaml list but not quoting it.

if i put in a yaml list (not shure why i should to that) i get the go %v representation of a list:

```yaml
        pre.hook.backup.velero.io/command: [sh -c /opt/bitnami/mariadb/bin/mysqldump -A -u root -p$MARIADB_ROOT_PASSWORD | /bin/gzip > /bitnami/mariadb/full_dump.gz]
```

Reply:
short update, i noticed while working with some other charts that they seem to use toYaml nowadays.

PR is updated  with and toYaml syntax.